Moderation-Policy.md

```diff
@@ -59,8 +59,8 @@ this default policy applies.
   permissions for an individual Collaborator's GitHub account from *all*
   Node.js GitHub Organization repositories as well as removing the account from
   the Node.js GitHub Organization membership.
-* *Ban* refers to the act of blocking an individual GitHub account from any
-  further participation in the Node.js GitHub Organization. Bans may be
+* *Blocks* refers to the act of blocking an individual GitHub account from any
+  further participation in the Node.js GitHub Organization. Blocks may be
```

Comment:
>**Block** refers to the act of blocking an individual GitHub account from any
   further participation in the Node.js GitHub Organization. **A block** may be

This probably should be singular.

Reply:
Thanks for the suggestion! Sounds good.